firedrake/cython/supermeshimpl.pyx

```diff
@@ -2,7 +2,7 @@
 
 import numpy
 from firedrake.petsc import PETSc
-from pyop2.datatypes import IntType, ScalarType
+from pyop2.datatypes import IntType, ScalarType, RealType
 
```

Comment:
Sometimes we import the types straight from PyOP2, sometimes we import them from firedrake.utils which imports them from PyOP2, sometimes from parameters. Maybe we should unify?

Reply:
I agree we should unify. I swaped imports from pyop2.datatypes with firedrake.utils. On the other hand parameters should be ok since they in most cases communicate scalar_type between TSFC?

Reply:
Note that we now import almost everywhere from `firedrake.utils`